python-checks/src/test/resources/checks/hotspots/graphQLIntrospection.py

```diff
@@ -87,6 +87,28 @@ def compliant_import():
         middleware=some_middleware, # if it is not a list or a tuple we should not raise an issue
     )
 
+    class IntrospectionCustomMiddleware:
+            ...
+    my_custom_middleware = IntrospectionCustomMiddleware()
+
+    GraphQLView.as_view(
+            name="introspection",
+            schema=schema,
+            graphiql=True,
+            middleware=[my_custom_middleware]
+        )
+
+    def create_middleware():
+        return IntrospectionCustomMiddleware()
+
+    # FP this should be fixed with SONARPY-1554
```

Comment:
I think we can just accept the FP without a ticket here: it doesn't look like something we'll really want to solve anytime soon.